### ISSUE TITLE
[REGRESSION!] TypeCast Unit TC Fail in aarch64/armv7l

### ISSUE THREAD

**Issue by myungjoo** [nnsuite/nnstreamer#882]
With the current master, 4ace9e923b873e90b1a097dff3a480f17c0d1d80

aarch64 / armv7l:
```
[  392s] ==================================================
[  392s]     Test Group transform_typecast Starts.
[  392s] [PASSED] 0:gst-launch of case 0
[  392s] [PASSED] 1:gst-launch of case 1
[  392s] [PASSED] 1:Golden test comparison
[  392s] [PASSED] 2:gst-launch of case 2
[  392s] [PASSED] 2:Golden test comparison
[  392s] [PASSED] 3:gst-launch of case 3
[  392s] [PASSED] 3:Golden test comparison
[  392s] [PASSED] 4:gst-launch of case 4
[  392s] [PASSED] 4:Golden test comparison
[  392s] [PASSED] 5:gst-launch of case 5
[  392s] [PASSED] 5:Golden test comparison
[  392s] [PASSED] 6:gst-launch of case 6
[  392s] [PASSED] 6:Golden test comparison
[  392s] [PASSED] 7:gst-launch of case 7
[  392s] [PASSED] 7:Golden test comparison
[  392s] [PASSED] 8:gst-launch of case 8
[  392s] [FAILED][Critical] 8:Golden test comparison
[  392s] [PASSED] 9:gst-launch of case 9
[  392s] [FAILED][Critical] 9:Golden test comparison
[  392s] ==================================================
[  392s] [FAILED] Test Group transform_typecast has failed cases (2)
[  392s] ==================================================
```

We definitely need nightly/daily unit test for armv7l & aarch64

**Comment by myungjoo** [comment]
Uh.. this is typecasting float-to-uint problem.
Didn't we resolve this already???
PTAL @wooksong

**Comment by wooksong** [comment]
@myungjoo Right. The typecasting float-to-uint issue was resolved and unit TCs #8/#9 were not failed until tensor_transform is refactored. Anyway, I will look into it.

While I am writing the comment, you've already submitted the PR to resolve this issue. Thanks.
